### PR TITLE
[discourse] Do not process topics with last_posted_at = None

### PR DESCRIPTION
**Pull request by acs**
This error is described at 

https://github.com/grimoirelab/perceval/issues/150

and it can be reproduced with:

`perceval discourse https://discourse.mozilla.org/`

This PR fixes it.

### CODE REVIEW

File: perceval/backends/core/discourse.py [hunk 1]
```diff
@@ -234,6 +234,9 @@ def __parse_topics_page(self, raw_json):
 
         for topic in topics_page['topic_list']['topics']:
             topic_id = topic['id']
+            # a topic without posts? - acs
```
Comment (sduenas) [inline perceval/backends/core/discourse.py]: What does this comment mean?

Comment (sduenas) [reply perceval/backends/core/discourse.py]: Before merging this, we need some unit tests.

Comment (acs) [reply perceval/backends/core/discourse.py]: About the comment, I am not totally sure why the topic does not have a last updated date, so my guess is that it has no posts yet. This is why I have left this comment.

I will work in the unit tests, yes!

**Comment by acs** [comment]
@sduenas I have removed the comment because there are posts in this topic with the null value for last_updated_at. So I have decided to log a warning about this fact.

**Comment by acs** [comment]
This issue is waiting for this fix:

https://github.com/mozilla/participation-metrics-org/issues/123

**Pull request merged**